README.md

```diff
@@ -117,6 +117,12 @@ so they look good.
 
 ## Usage
 
+### Format all Ruby files in a directory and all subdirectories (recursive)
+
+```
+$ rufo .
+```
```

Comment:
Would the readme be clearer if this was added to the  `format files or directories` section below?

Something like:
```
# All directories
$ rufo . 

# Specific directory
$ rufo [file or directory name]
```

Reply:
Sure, updated!